content/en/docs/KRE/user/50_krt_v2/specs/index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "KRT V2 YAML specs"
+description: >
+  YAML Definition of a KRT file.
+weight: 20
+---
+
+A KRT yaml file is a declarative file describing the content of the KRT.
+It has general description of the Version, a GRPC entrypoint for the Runtime Version and
+nodes that are connected between each other to form workflows that will be access through GRPC services defined on the entrypoint.  
+
+## Fields Specs
+
+Here is a description of each field divided in five main concepts that make a KRT file:
+
+### Metadata
+
+Descriptive information of the Runtime Version.
+
+- **version**: the unique ID of this version.
```

Comment:
Any restrictions if any. For example, version can't have whitespaces

Reply:
That's true, there are indeed restrictions, I'll add them.